paimon-flink/paimon-flink-common/src/test/java/org/apache/paimon/flink/action/SortCompactActionForUnawareBucketITCase.java

```diff
@@ -287,14 +293,21 @@ private SortCompactAction createAction(String orderStrategy, List<String> column
                 "--order_strategy",
                 orderStrategy,
                 "--order_by",
-                String.join(",", columns));
+                String.join(",", columns),
+                "--sort_conf range_strategy=" + rangeStrategy,
```

Comment:
We can add this as table-config, when we do compaction, we just --table_conf sort.range.strategy = xxx. By this way, we can simplify code and keep compatibility with old code.